udapi/core/bundle.py

```diff
@@ -7,7 +7,12 @@
 from root import Root
 
 class Bundle(object):
-    """Bundle can be used for embracing two or more Universal Dependency trees that are associated in some way (e.g. parallel translations) inside a document. Unless different zones are differentiated in a bundle, there's only one tree per bundle by default."""
```

Comment:
PEP008 says that for docstrings (and comments), the line length should be limited to 72 characters (and for other code 79 characters).
I would say that even 100 characters is ok (unless we want to strictly follow PEPs), but this line has 262 characters.